yarn-project/ethereum/src/config.ts

```diff
@@ -175,7 +176,7 @@ const DefaultRewardConfig = {
   sequencerBps: 5000,
   rewardDistributor: EthAddress.ZERO.toString(),
   booster: EthAddress.ZERO.toString(),
-  blockReward: BigInt(50e18),
+  blockReward: 50n * 10n ** 18n,
```

Comment:
The value in [TMNT-263: Rollup Configuration](https://linear.app/aztec-labs/issue/TMNT-263/rollup-configuration) is 500.